opslevel/datasource_opslevel_group.go

```diff
@@ -52,6 +52,22 @@ func datasourceGroup() *schema.Resource {
 					},
 				},
 			},
+			"teams": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
```

Comment:
Now that I've implemented the resource, it seems like we would want this to be a list similar to what's in the resource schema instead of a map.  Can't remember why we decided to do maps for parent and members though.  I feel like I would want to just reference data.opslevel_group.foo.teams and it would be the same type as what the resource is expecting, e.g. list of string aliases.

Reply:
Ya that would be a good improvement.